examples/hackernews/back-end/reactive_service/package.json

```
@@ -10,7 +10,7 @@
   "license": "MIT",
   "description": "",
   "dependencies": {
-    "@skipruntime/core": "^1.0.0",
+    "skip-wasm": "^1.0.0",
```

```suggestion
    "@skipruntime/core": "npm:skip-wasm@^1.0.0",
```

This way, we don't need to change the downstream code.

---

This does not seem to work in the current development chain.

---

What error(s) are you getting?

---

Oh yeah, this probably clashes with the workspace thing. There might be a way to make it work though, did you try removing the ˋnpm:` prefix?

---

The skip-wasm@^1.0.0 does not exists.

---

Removing `npm:` `skip-wasm@^1.0.0 ` considered as version number

---

pnpm seems to manage it with `workspace:` prefix, but not npm.